xwiki-rendering-syntaxes/xwiki-rendering-syntax-xhtml/src/test/resources/xhtml10/specific/paragraph/paragraph.test

```diff
@@ -0,0 +1,10 @@
+.#-----------------------------------------------------
+.input|xhtml/1.0
```

Comment:
a better solution is to use `inputexpect`. This will test is both ways and is shorter to write.

Reply:
Right now in your test you don't using xhtml as input.